python/api/openapi.yaml

```diff
@@ -1095,7 +1095,7 @@ paths:
                 type: array
                 items:
                   type: string
-                example: ["started", "inProgress", "finish"]
+                  enum: ["started", "inProgress", "finish"]
```

Comment:
This endpoint is used to retrieve the possible values. It will be soon fixed in the openapi (then we can delete this endpoint).